packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -177,6 +179,10 @@ export async function getSenderReceiverClients(
   senderClient: QueueClient | TopicClient;
   receiverClient: QueueClient | SubscriptionClient;
 }> {
+  let queueName: string;
+  let topicName: string;
+  let subscriptionName: string;
+  let queueClient: QueueClient;
```

Comment:
Why have them as variables here at all? They don't really get re-used.
I would suggest to have them as `const` inside each `case` statement.